README.md

```diff
@@ -14,30 +14,36 @@ an exact solution if possible or use an appropriate numeric solver otherwise.
+This will download all required packages and libraries and create a container that uses the pre-built version of the [latest release of NESTML](https://github.com/nest/nestml/releases). If you are interested in using the bleeding edge version of NESTML (a.k.a. Git master), you can add the argument `--dev` to the invocation of the `./nestml_container.sh` script.
+
+If everything goes well, the list printed by the command 'docker images' should now contain the 'nestml_release' container. If you experience an error, please [open an issue](https://github.com/nest/nestml/issues) so we can look into and fix it.
+
+### Running
+
+To actually convert your model files written in NESTML to NEST C++, you have to run the Docker container. This is again done using the `nestml_container.sh` script, which for this purpos gets the command `run` as first argument and one or more folders containing one or more `.nestml` files (the folder is called `<models>` in the following description):
+
 ```
-sh nestml_docker.sh run ~/testing
+./nestml_container.sh run <models>
 ```
 
-It creates a subfolder 'build' in the 'testing' folder that contains the generated code. Per default, the module for the generated stuff is called 'testing', e.g. the name of the input folder. In order to integrate the model into nest, use the following commands (assumption, you switched to the 'testing/result'):
+This run creates a subfolder `build` in the `<models>` directory that contains the generated code and all infrastructure and source files for an extension module for NESTML, which can be dynamically loaded. The module will have the same name as the folder in which you stored the `.nestml` files. In order to compile and install the module, use the following commands:
 ```
-cmake -Dwith-nest=${NEST_INSTALL_DIR}/bin/nest-config .
+cd <models>/build
+cmake -Dwith-nest=<nest_install_dir>/bin/nest-config .
+make all
 make install
 ```
-Again, if everything goes well, you can use the generated module with neuron models in NEST and PyNEST Scripts. E.g.
-```
-nest.Install("testing")
-neuron=nest.Create ("iaf_cond_alpha_implicit")
-```
+
+Again, if everything goes well, you can now use the generated module in your SLI and PyNEST scripts by using the corresponding version of the `Install` command. For SLI the invocation looks like this: `(<models>) Install`, for PyNEST it reads `nest.Install("<models>")`. After loading the module, the contained models can be instantiated just as the built-in models using the `Create` command in SLI and PyNEST, respectively.
```

Comment:
I think would appreciate, if only "real" models will be used in these examples. For example, the path to the git models can be used in the example. Then,  
```
nest.Install("models")
iaf_psc_alpha = nest.Create("iaf_psc_alpha_neuron")
iaf_psc_exp = nest.Create("iaf_psc_exp_neuron")
```